include/aspect/utilities.h

```diff
@@ -47,6 +47,15 @@ namespace aspect
     using namespace dealii;
     using namespace dealii::Utilities;
 
+
+    /**
+     * Split the set of DoFs (typically locally owned or relevant) in @p whole_set into blocks
+     * given by the @p dofs_per_block structure.
+     */
+    void split_by_block (const std::vector<types::global_dof_index> &dofs_per_block,
+                         const IndexSet &whole_set,
```

Comment:
say something like "The numbers of dofs per block need to add up to the size of the index space described by the second argument."